client/modules/User/components/Collection.jsx

```diff
@@ -56,11 +57,11 @@ const ShareURL = ({ value }) => {
         onClick={() => setShowURL(!showURL)}
         iconAfter={<DropdownArrowIcon />}
       >
-        Share
+        {t('Collection.ShareURL_Share')}
```

Comment:
Why the introduction of the underscore in the translation keys? As someone looking at this, it's make me wonder if these are special since all the keys I've seen are like this: `Collection.ShareURLShare`